packages/chat/.dependency-cruiser.mjs

```diff
@@ -1,3 +1,12 @@
 import { createConfig } from '@voiceflow/dependency-cruiser-config';
 
-export default createConfig({ allowTypeCycles: true });
+export default createConfig({
+  allowTypeCycles: true,
+  orphans: {
+    ignore: [
+      'src/components/VoiceWidget/WaveformVisualizer.component.tsx',
+      'src/components/VoiceWidget/MovingGradient.component.tsx',
+      'src/components/VoiceWidget/MockVoiceWidgetImage.tsx',
+    ],
```

Comment:
I am not sure that this is permanently needed, however those files seem to be useful but not used (yet or already)